src/mockApi/getData.js

```diff
@@ -0,0 +1,53 @@
+import { baseURL } from '../shared/constants';
+import { filterCurrentTasks, filterProgress, filterMembers } from '../shared/helpers';
+
+export async function getFakeUsers() {
+  const response = await fetch(`${baseURL}/users`);
+  const users = await response.json();
+
+  return filterMembers(users);
+}
+
+export async function getFakeTasks() {
+  const response = await fetch(`${baseURL}/tasks`);
+  const tasks = await response.json();
+
+  return tasks;
+}
+
+export async function getMemberTasks({ id }) {
+  const response = await fetch(`${baseURL}/users/?id=${id}`);
+  const user = await response.json();
+  const { tasksId } = user[0];
+  const tasks = await Promise.all(
+    tasksId.map(async (taskId) => {
+      const currentTask = await fetch(`${baseURL}/tasks/${taskId}`);
+      const task = await currentTask.json();
+
+      return task;
+    }),
+  );
+
+  return filterCurrentTasks(tasks);
+}
+
+export async function getTraks(userId) {
+  const memberId = await userId;
```

Comment:
what's about this place ? you don't need `await` here too

Reply:
Yes, await not needs here. Thanks. Removed.